crates/astria-sequencer-types/src/namespace.rs

```diff
@@ -41,7 +41,7 @@ impl Namespace {
     /// returns an 10-byte namespace given a byte slice by hashing
     /// the bytes with sha256 and returning the first 10 bytes.
     #[must_use]
-    pub fn from_slice(bytes: &[u8]) -> Namespace {
+    pub fn with_hashed_bytes(bytes: &[u8]) -> Namespace {
```

Comment:
Strictly speaking this does not have anything to do with the rest, but it bugged me because `from_slice` is much more innocent than what actually happens. Also this brings naming in line with `ChainId::with_hashed_bytes`